rlang/pom.xml

```diff
@@ -38,7 +38,7 @@
 
         <spark.archive>spark-${spark.version}</spark.archive>
         <spark.bin.download.url>
-            https://archive.apache.org/dist/spark/${spark.archive}/${spark.archive}-bin-without-hadoop.tgz
+            https://archive.apache.org/dist/spark/${spark.archive}/${spark.archive}-bin-hadoop3.tgz
```

Comment:
Why this change? As I see it, only R libraries are needed and extracted from the download.

Reply:
we have `SparkRBackend` and `RInterpreterTest#testSparkRInterpreter`.

Spark core requires Hadoop client classes to run, previously, we forcibly shipped the Hadoop classes into the Rlang interpreter, IMO this is not the right approach, the Spark runtime should handle Hadoop classes.

Reply:
I think you have not understood my comment or I have overlooked a place where the files are used.
In my opinion, the downloaded files are only used in the Maven execution maven-resources-plugin:copy-sparkr-files.
Only a small fraction of the files are copied here. Each file under `/R/lib`. This folder is contained in both `spark-bin-hadoop3.tgz` and `spark-bin-without-hadoop.tgz`.

I don't see the hadoop libraries from this download being used anywhere.

Reply:
@Reamer You are right, the CI result looks good.